qa/tasks/cephfs/test_nfs.py

```diff
@@ -70,20 +70,41 @@ def _disable_nfs(self):
         log.info("Disabling NFS")
         self._sys_cmd(['sudo', 'systemctl', 'disable', 'nfs-server', '--now'])
 
-    def _fetch_nfs_status(self):
-        return self._orch_cmd('ps', f'--service_name={self.expected_name}')
+    def _fetch_nfs_daemons_details(self, enable_json=False):
+        args = ('ps', f'--service_name={self.expected_name}')
+        if enable_json:
+            args = (*args, '--format=json')
+        return self._orch_cmd(*args)
+
+    def _check_nfs_cluster_event(self, expected_event):
+        '''
+        Check whether an event occured during the lifetime of the NFS service
+        :param expected_event: event that was expected to occur
+        '''
+        # Wait few seconds for NFS daemons' status to be updated
+        wait_time = 10
+        while wait_time <= 120:
```

Comment:
While you are here, could this be changed to use `safe_while`?

Reply:
Done